ClangPowerTools/ClangPowerTools/Commands/CommandController.cs

```diff
@@ -402,6 +412,13 @@ public void OnBeforeClangCommand(object sender, EventArgs e)
       if (!(sender is OleMenuCommand command))
         return;
 
+      if (areCommandsDisabled)
+      {
+        command.Enabled = false;
+        return;
+      }
+      command.Enabled = true;
```

Comment:
Considering the below checks it's necessary to activate the command at this point?

Reply:
I just wanted to make sure that commands are enabled when they should be. Are they enabled by default when you open a solution?

Reply:
Yes. Check the `OnBeforeClangCommand()` method from `CommandController` class.